redhat-director-open-source-program -office.md

```diff
@@ -0,0 +1,27 @@
+# Director Open Source Program Office (RedHat)
```

Comment:
Note (minor nit) should be Red Hat rather than RedHat. (My former colleagues in Red Hat's branding team will thank you.)